cudaaligner/src/aligner_global_myers_banded.cpp

```diff
@@ -278,12 +290,20 @@ StatusType AlignerGlobalMyersBanded::align_all()
     {
         result_lengths_d.clear_and_resize(n_alignments);
     }
+    if (get_size(scheduling_index_d) < n_alignments)
+    {
+        scheduling_index_d.clear_and_resize(n_alignments);
+    }
+    scheduling_index_h.resize(n_alignments);
+    std::iota(begin(scheduling_index_h), end(scheduling_index_h), 0);
+    std::sort(begin(scheduling_index_h), end(scheduling_index_h), [&seq_starts_h](int32_t i, int32_t j) { return seq_starts_h[2 * i + 2] - seq_starts_h[2 * i] > seq_starts_h[2 * j + 2] - seq_starts_h[2 * j]; });
```

Comment:
Add a comment

Reply:
Yes, you're right. This needs a comment. Done.